qtmodern/windows.py

```diff
@@ -146,8 +146,10 @@ def __child_was_closed(self):
 
     def closeEvent(self, event):
         if not self._w:
-            return True
-        event.setAccepted(self._w.close())
+            event.accept()
+        else:
+            self._w.close()
+            event.setAccepted(self._w.isHidden())
```

Comment:
isHidden seems to be the most reliable way to figure out if a window is closed or not